apps/user/services/authentication.go

```diff
@@ -50,6 +54,47 @@ func (service AuthenticationService) Login(c *gin.Context) map[string]string {
 	}
 }
 
+// BasicLogin ...
+func (service AuthenticationService) BasicLogin(c *gin.Context) (
+	userResponse *userSerializers.UserAuthSerializer,
+	status int,
+	err error) {
+
+	var userData userSerializers.UserLogin
+	err = c.BindJSON(&userData)
+	if err != nil {
+		return nil, http.StatusInternalServerError, err
+	}
+
+	gormDB := service.DB
+	userResponse = new(userSerializers.UserAuthSerializer)
+
+	err = gormDB.Model(&userModels.User{}).
+		Where("email = ?", userData.Email).
+		Scan(&userResponse).Error
+	if err != nil {
+		if err == gorm.ErrRecordNotFound {
+			return getInvalidEmailPasswordErrorResponse()
+		}
+		return getInternalErrorResponse()
+	}
+	encryptedPassword := EncryptPassword(userData.Password)
+	if !reflect.DeepEqual(encryptedPassword, userResponse.Password) || userResponse.Password == nil {
+		return getInvalidEmailPasswordErrorResponse()
+	}
+
+	session := sessions.Default(c)
+	userResponse.Token = utils.RandToken()
+	startSession(session, userResponse)
+	return userResponse, http.StatusAccepted, nil
+}
+
+// EncryptPassword ...
+func EncryptPassword(password string) []byte {
+	encryptedPassword := pbkdf2.Key([]byte(password), nil, 100000, 256, sha256.New)
```

Comment:
```suggestion
	return pbkdf2.Key([]byte(password), nil, 100000, 256, sha256.New)
```

Reply:
Please make constants for the constant values, 100000, 256, etc. and add proper comments.

Reply:
done.